keyboards/synthlabs/060/060.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 Aaron Hong (@hongaaronc)
```

Comment:
This file should be removed.

Reply:
Removed as of https://github.com/qmk/qmk_firmware/pull/19363/commits/ac78af58d935e88d710a364235552821d32e5543.